recipes/uldaq/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "uldaq" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5cfc152a67381717bcf6f275f4f059c7adfae830eb4dfca35d90972a6021f84c
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
This can be a `noarch: python` recipe

Reply:
I add this because underlying library, `libuldaq`, can only be used on macOS and linux.

Reply:
That's OK. The environment on windows just won't be able to solve.  It is better for the package to be noarch

Reply:
Sounds good. Fixed.